deps/wazuh_testing/wazuh_testing/modules/fim/__init__.py

```diff
@@ -133,6 +128,9 @@
 ERR_MSG_INTEGRITY_OR_WHODATA_NOT_STARTED = 'Did not receive expected "File integrity monitoring real-time Whodata \
                                             engine started" or "Initializing FIM Integrity Synchronization check"'
 ERR_MSG_INTEGRITY_CHECK_EVENT = 'Did not receive expected "Initializing FIM Integrity Synchronization check" event'
+ERR_MSG_CONTENT_CHANGES_EMPTY = "content_changes is empty"
+ERR_MSG_CONTENT_CHANGES_NOT_EMPTY = "content_changes isn't empty"

```

Comment:
This is an error message that will not change because it is not variable. It just states the results found (empty or not) Extracting the word `content_changes` to another variable seems like over-engineering the message